packages/react-components/src/button.tsx

```diff
@@ -81,6 +80,7 @@ const smallStyles = css({
   '> span + svg': {
     marginLeft: `${6 / perRem}em`,
   },
+  alignItems: 'center',
```

Comment:
It seems strange that large and small should differ in this way

Reply:
This keeps the same behaviour. I can set the line height lower and remove the height restriction for it to wrap the same way as the line style if you prefer